internal/guest/runtime/hcsv2/uvm.go

```diff
@@ -549,9 +566,22 @@ func (h *Host) modifyHostSettings(ctx context.Context, containerID string, req *
 		return modifySCSIDevice(ctx, req.RequestType, req.Settings.(*guestresource.SCSIDevice))
 	case guestresource.ResourceTypeMappedVirtualDisk:
 		mvd := req.Settings.(*guestresource.LCOWMappedVirtualDisk)
+
+		if mvd.ContainerMount {
+			// Ensure that devices for container mounts are always mounted at /run/mounts/scsi/m.
+			vhdMountPathCompiledExpression := regexp.MustCompile(guestpath.LCOWSCSIMountPrefixRegex)
```

Comment:
I think we can make a `vhdMountPathCompiledExpression` a var and compile the regex only once and just reuse that with each request.